src/main/java/uk/gov/hmcts/reform/professionalapi/controller/OrganisationController.java

```diff
@@ -82,4 +94,29 @@
                 .status(200)
                 .body(organisationResponse);
     }
-}
+
+    @ApiOperation("Retrieves an organisations payment accounts by super user email")
+    @ApiResponses({
+            @ApiResponse(
+                    code = 200,
+                    message = "The organisations associated payment accounts",
+                    response = OrganisationPbaResponse.class
+            )
+    })
+    @GetMapping(path = "/pbas")
```

Comment:
Is this path correct?  i was under impression we are doing /organisation/pbas?<query param>

Reply:
As this is part of OrganisationController.java the path must begin with "v1/organisations" (line 31) and so the GET request for this ticket will be made with "/pbas" (line 106) added onto the end of the above path... as a result the overall path will be: "v1/organisations/pbas".